server/main.lua

```diff
@@ -188,24 +188,51 @@ AddEventHandler("playerDropped", function()
 	end
 end)
 
+local flags = {}
+
 RegisterNetEvent('hospital:server:RevivePlayer', function(playerId, isOldMan)
 	local src = source
 	local Player = QBCore.Functions.GetPlayer(src)
 	local Patient = QBCore.Functions.GetPlayer(playerId)
 	local oldMan = isOldMan or false
 	if Patient then
-		if oldMan then
-			if Player.Functions.RemoveMoney("cash", 5000, "revived-player") then
+		if Player.PlayerData.job.name == "ambulance" or QBCore.Functions.HasItem(src, "firstaid", 1) then
+			if oldMan then
+				if Player.Functions.RemoveMoney("cash", 5000, "revived-player") then
+					Player.Functions.RemoveItem('firstaid', 1)
+					TriggerClientEvent('inventory:client:ItemBox', src, QBCore.Shared.Items['firstaid'], "remove")
+					TriggerClientEvent('hospital:client:Revive', Patient.PlayerData.source)
+				else
+					TriggerClientEvent('QBCore:Notify', src, Lang:t('error.not_enough_money'), "error")
+				end
+			else
 				Player.Functions.RemoveItem('firstaid', 1)
 				TriggerClientEvent('inventory:client:ItemBox', src, QBCore.Shared.Items['firstaid'], "remove")
 				TriggerClientEvent('hospital:client:Revive', Patient.PlayerData.source)
-			else
-				TriggerClientEvent('QBCore:Notify', src, Lang:t('error.not_enough_money'), "error")
 			end
 		else
-			Player.Functions.RemoveItem('firstaid', 1)
-			TriggerClientEvent('inventory:client:ItemBox', src, QBCore.Shared.Items['firstaid'], "remove")
-			TriggerClientEvent('hospital:client:Revive', Patient.PlayerData.source)
+			local license = Player.PlayerData.license
+			if flags[license] then
+				flags[license] = flags[license] + 1
+			else
+				flags[license] = 1
+			end
+			if flags[license] >= 3 then
+				MySQL.insert('INSERT INTO bans (name, license, discord, ip, reason, expire, bannedby) VALUES (?, ?, ?, ?, ?, ?, ?)',
+					{
+						GetPlayerName(src),
+						QBCore.Functions.GetIdentifier(src, 'license'),
+						QBCore.Functions.GetIdentifier(src, 'discord'),
+						QBCore.Functions.GetIdentifier(src, 'ip'),
+						"Trying to revive theirselves or other players",
+						2147483647,
+						'qb-ambulancejob'
+					}
+				)
+				TriggerEvent('qb-log:server:CreateLog', 'ambulancejob', 'Player Banned', 'red',
+					string.format('%s was banned by %s for %s', GetPlayerName(src), 'qb-ambulancejob', "Trying to revive theirselves or other players"), true)
+				DropPlayer(src, 'You were permanently banned by the server for: Exploiting')
```

Comment:
```suggestion
				exports['qb-core']:ExploitBan(src, 'Attempted exploiting RevivePlayer event')
```
Whole chunk of code can be replaced by existing qb-core export.